en/docs/learn/fraud-detection.md

```diff
@@ -0,0 +1,422 @@
+# Fraud Detection, Risk Based Authentication and Identity Verification
+
+## Overview
+
+### Fraud Detection
+
+Fraud detection is a set of activities undertaken to prevent information from being acquired under false pretences.
+Fraud detection can be applied to many industries such as banking, insurance. In banking, fraud may include forging
+checks or credit card thefts. When it comes to identity domain, fraudulent activities relate to identity thefts or
+system intrusions. A system intrusion is an intentional attempt to access/modify unauthorized information in the
+system and convert the system to an unreliable state through information leakage or denial of service attacks.
+
+Fraud detection involves monitoring the behavior of users in order to estimate anomalous behavior. These behaviors
+can have multiple facets including fraud, intrusion, account defaulting and delinquency. At present fraud detection
+has been implemented by utilizing various methods such as data mining, statistics, and artificial intelligence. All
+of these methods analyze patterns and data about users to identify and prevent suspicious behaviors.
+
+### Risk-Based Authentication
+
+Password-based authentication is still one of the primary authentication mechanisms in most of the enterprise systems
+. It is highly insecure and is prone to phishing attacks. Therefore organizations should enforce additional security
+measures to ensure the security of their users. Multi-factor authentication is a widely accepted mechanism to enhance
+ the security of user authentication. However, it is not a popular choice among users due to the negative impact it
+ has on user experience. Risk based authentication (RBA)  is another mechanism that provides secure authentication
+ while having a minimal impact on user experience. Furthermore, it ensures protection from security risks such as
+ password database leaks, credential stuffing, insecure passwords and guessing attacks.
+
+RBA is an adaptive authentication mechanism which applies different authentication levels to a user, based on a risk
+factor associated with the user. This risk factor can be used as a deciding point when predicting the probability of
+a system compromise. As the risk increases, the process of authentication should become more strict and comprehensive.
+
+RBA systems work by analysing additional properties of user us which are contextually available during login. These
+include the device of login, geolocation, IP address etc. Based on these properties a risk score is calculated for
```

Comment:
include the login device, geolocation, IP address, etc. Based on these properties,